<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookmod</class>
 <widget class="QDialog" name="bookmod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>书名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leBname">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWriter">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>热度：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leHot">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbLend">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>可借</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbNlend">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>116</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>不可借</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSum">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCon">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>重填</string>
   </property>
  </widget>
  <widget class="QLabel" name="lberror">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Sorry,  data   load    error !!!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
